lib/offsite_payments/integrations/moneybookers.rb

```diff
@@ -174,23 +214,32 @@ def secret
           @options[:credential2]
         end
 
-        # Acknowledge the transaction to MoneyBooker. This method has to be called after a new
-        # apc arrives. It will verify that all the information we received is correct and will return a
-        # ok or a fail. The secret (second credential) has to be provided in the parameter :credential2
-        # when instantiating the Notification object.
+        # Acknowledge the transaction to MoneyBooker.
+        # This method has to be called after a new apc arrives.
+        # It will verify that all the information we received is correct
+        # and will return a ok or a fail.
+        # The secret (second credential) has to be provided in the parameter
+        # :credential2 when instantiating the Notification object.
```

Comment:
max line width is set at 120 char for this project, no need to modify the comments